src/lmic/lmic_eu868.c

```diff
@@ -236,7 +236,7 @@ ostime_t LMICeu868_nextTx(ostime_t now) {
 #if !defined(DISABLE_BEACONS)
 void LMICeu868_setBcnRxParams(void) {
         LMIC.dataLen = 0;
-        LMIC.freq = LMIC.channelFreq[LMIC.bcnChnl] & ~(u4_t)3;
+        LMIC.freq = FREQ_BCN;
```

Comment:
We need to implement `MCMD_BeaconFreqReq` at some point. At that point, the `FREQ_BCN` can be overridden. We can introduce a new class B data structure to hold the beacon frequency. We should remove `CHAN_BCN` from the For the US, we have to deal with channel-hopping beacons. The channel numbers assigned in `lorabase.h` should not be used, because those entries in `LMIC.channels[]` are used for other things and can't be safely used for the classB beacon. Since we already have `LMIC.ping.*`, we could possibly add it there, or simply add `LMIC.beacon.freq` etc.

Reply:
Added first stage (static beacon frequency) for AS923, EU868, IN866 and KR920 as part of major changes for #208.